api_docs/include/rest-endpoints.md

```diff
@@ -38,8 +38,9 @@
 * [Get subscribed channels](/api/get-subscriptions)
 * [Subscribe to a channel](/api/subscribe)
 * [Unsubscribe from a channel](/api/unsubscribe)
+* [Update subscriptions](/api/update-subscriptions)
```

Comment:
This one should probably be "Bulk update channel subscriptions". I'm not entirely convinced this endpoint needs to exist, to be honest, let's discuss that a bit before adding it to the visible documentation.

Reply:
Removed for now.